src/SamplesApp/UITests.Shared/Windows_UI_ViewManagement/IsScreenCaptureEnabledTests.xaml.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Uno.UI.Samples.Controls;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.ViewManagement;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+
+namespace UITests.Shared.Windows_UI_ViewManagement
+{
+	[SampleControlInfo("Windows.UI.ViewManagement", "IsScreenCaptureEnabled", description: "Allows disabling screen capture (currently available on Android only)")]
+	public sealed partial class IsScreenCaptureEnabledTests : UserControl
+    {
+        public IsScreenCaptureEnabledTests()
+        {
+            this.InitializeComponent();
+#if __ANDROID__ || WINDOWS_UWP
```

Comment:
Use `ApiInformation.IsPropertyPresent("Windows.UI.ViewManagement.ApplicationView", "IsScreenCaptureEnabled")` instead, so the test is automatically enabled when the feature gets implemented.

Reply:
Done :-)